packages/webpack-config-anansi/src/base/index.js

```diff
@@ -89,6 +90,33 @@ export default function makeBaseConfig({
     ];
   }
 
+  const plugins = [
+    new StatsWriterPlugin({
+      filename: manifestFilename,
+      stats: {
+        chunkModules: false,
+        source: false,
+        chunks: false,
+        modules: false,
+        assets: true,
+      },
+    }),
+  ]
+
+  if(cssExtractOptions !== false) {
+    plugins.push(new MiniCssExtractPlugin({
+      filename:
+        (mode !== 'production') | nohash
+          ? '[name].css'
+          : '[name].[contenthash].css',
+      chunkFilename:
+        (mode !== 'production') | nohash
+          ? '[name].css'
+          : '[name].[contenthash].css',
+      ...cssExtractOptions,
```

Comment:
if these are the same options as what is sent to the loader, why are my defaults different? I think the loader and plugin options are different things

Reply:
I couldn't tell you, I just moved the existing code around

Reply:
I've made the update to only apply the options to the plugin, and use the falsiness to include (or not) the loader.